service/service-samples/sample-service-reactive-oracle-book/src/main/java/io/americanexpress/service/book/rest/utils/BookServiceMapper.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 American Express Travel Related Services Company, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.americanexpress.service.book.rest.utils;
+
+import io.americanexpress.data.oracle.book.entity.BookEntity;
+import io.americanexpress.service.book.rest.model.CreateBookRequest;
+import io.americanexpress.service.book.rest.model.CreateBookResponse;
+import io.americanexpress.service.book.rest.model.ReadBookResponse;
+import org.apache.commons.lang3.StringUtils;
+
+public class BookServiceMapper {
```

Comment:
delete the utils folder and move it this class under service.helper package

Reply:
split the class in a single according to the single responsibility principle SRE ,  and make the method static with a private constructor so that you can call this as an example
CreateBookEntityCreator.create().
refer to other modules for example